Client Scripts/Enable VIP and Senior VIP Checkboxes and read only/ScriptToEnableVIP_superVIP users

```diff
@@ -0,0 +1,11 @@
+function onLoad() {
+   //Type appropriate comment here, and begin script below
+   var myRole= g_user.hasRole("admin");
+	if(myRole!=true){
+		g_form.setReadOnly("vip",true);
+		g_form.setReadOnly("u_senior_vip",true);
+	}else{
+		g_form.setReadOnly("vip",false);
+		g_form.setReadOnly("u_senior_vip",false);
```

Comment:
´u_senior_vip´ is not an out of the box field and there the code will generate errors for most users. I recommend removing it and relying on standard fields only.